cmlreaders/base_reader.py

```diff
@@ -0,0 +1,87 @@
+from typing import Optional
+
+from .path_finder import PathFinder
+from .exc import UnsupportedOutputFormat
+from .cmlreader import CMLReader
+
+
+class MetaReader(type):
+    """ Metaclass for all CML readers
+
+        The responsibility of the metaclass is to register the reader with
+        the generic CMLReader to avoid needing to do this as a manual step.
+
+    Notes
+    -----
+    When a child-classes uses `cmlreaders.base_reader.MetaReader` as a meta
+    class, the reader_names dictionary stored as a class variable in
+    `cmlreaders.cmlreader.CMLReader` is updated based on the `data_types`
```

Comment:
Single backticks aren't the same thing in RST as they are in Markdown. You probably want to use double backticks here.